server/database/dummyDataERD.js

```diff
@@ -0,0 +1,43 @@
+export async function insertERDiagramsData(database) {
```

Comment:
`nanoid` is undefined here:

```suggestion
import { nanoid } from 'nanoid';

export async function insertERDiagramsData(database) {
```

Reply:
fixed